tests/spec/core/webidl-contiguous-spec.js

```diff
@@ -37,7 +37,13 @@ describe("Core - Contiguous WebIDL", function () {
             expect($target.text()).toEqual(text);
             expect($target.find(".extAttr").text()).toEqual("Something");
             expect($target.find(".idlCtor").text()).toEqual("Constructor()");
-
+            
+            $target = $("#if-identifier-list", doc);
+            text = "[Global=Window,\n Exposed=(Window,Worker)]\ninterface SuperStar {\n};";
```

Comment:
Seems the common practice is indeed not to use spaces (e.g.: [ServiceWorker](https://slightlyoff.github.io/ServiceWorker/spec/service_worker/index.html#service-worker-global-scope) and [WebWorker](http://dev.w3.org/html5/workers/#dedicated-workers-and-the-dedicatedworkerglobalscope-interface) specs).